table/block_based/block_based_table_factory.cc

```diff
@@ -454,6 +454,7 @@ void BlockBasedTableFactory::InitializeOptions() {
     // It makes little sense to pay overhead for mid-point insertion while the
     // block size is only 8MB.
     co.high_pri_pool_ratio = 0.0;
+    co.low_pri_pool_ratio = 1.0;
```

Comment:
Question: I see that in LRUCache, the default of low_pri_pool_ratio is 0.5, while it is 1.0 here. Am I missing anything?

Reply:
Maybe use a constant to represent the default value?

Reply:
@riversand963 's comment just made me realize that we would probably want to adjust the default of `low_pri_pool_ratio` for the purposes of backward compatibility. That is, I think what we would want is a default of say, -1, where negative values would be interpreted as "use (1 - high_pri_pool_ratio)". With the current default of 0.5, we run the risk of applications hitting errors during cache creation if they set high_pri_pool_ratio to a value greater than 0.5.

Reply:
Alternatively, we could make negative values mean "unlimited low-priority pool capacity."

Side note: it would make sense for the pool capacity data members to be `size_t`s as opposed to `double`s.

Reply:
Will do.